readme.md

```diff
@@ -517,6 +519,12 @@
 - [Blockchain AI](https://github.com/steven2358/awesome-blockchain-ai#readme) - Blockchain projects for artificial intelligence and machine learning.
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 
+
```

Comment:
Again two lines here?

Reply:
If you see other sections, they have two lines.